solution/flask_apps/models/Admissions.py

```diff
@@ -0,0 +1,88 @@
+from ..services.db.db_handler import PostgresDBService
+import psycopg2
+
+import pandas as pd
+
+from .AbstractModel import AbstractModel
+
+
+class Admissions:
+
+    def __init__(self, person_id, class_id):
+        self.id = None
+        self.person_id = person_id
+        self.class_id = class_id
+
```

Comment:
I would be better is `ModelSql` was a manger for a model for example
```python
class Manager(AbstractManager):
    def save(self,*args,**kwargs)->Model:
          #impement
    def get_id(self,id)->Model:
          #implement
class Model(AbstractModel):
    query = Manager()
     def ___init__(self,field1):
         self.field1=field1
m = Model.query.get_id(1)
m = Model.query.save(field1='x')
```
so that you dont have to always import `Model` and `ModelSql`, seems too much work.